testing/resources/containers/cb-with-exporter.Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Change
+ARG couchbase_server_image=couchbase/server:enterprise-6.6.3
+
+# Build the exporter
+FROM golang:1.17.2 as go_build
+RUN git clone https://github.com/couchbase/couchbase-exporter.git /opt/couchbase-exporter
```

Comment:
probably want to limit the depth here, do we want to support a gitref arg as well to get a specific commit, etc.?

Reply:
Both are good points, but AIUI they're mutually exclusive - we can either use depth=1, or clone the whole repo and check out a specific commit. (You *can* clone a specific branch, but that won't work for commits.) cf. https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---branchltnamegt

Reply:
Agreed, go with a deep clone but add support for a gitref. That could be another JIRA though - if so then make sure it is just a shallow clone here.